### PR TITLE
Please review initial draft. Let's discuss in comments

### PR DESCRIPTION
**Pull request by dreinavarro**

### CODE REVIEW

File: guideline-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+### Guidelines for adding/modifying these Guidelines
+
+1. Create a topic branch for the proposed addition/change
+2. If adding a new file use `lisp-case` i.e. `new-file.md`
+3. Create a pull-request for change
+4. Change must be reviewed before it can be merged to `master`
```
Comment (dreinavarro) [inline guideline-guidelines.md]: I think point #4 is redundant

**Review by dreinavarro — COMMENTED** [review]
Added my own comments

**Comment by dreinavarro** [comment]
Please provide your comments

**Review by alecguintu — APPROVED** [review]
Looks good.

**Pull request merged**